### PR TITLE
added new option in order to run server from specific folder

### PR DESCRIPTION
**Pull request by nick-bogdanov**
Currently I'm working under my project which are deployed to heroku.
In order to start server on heroku I need to serve my dist folder.
I found this option helpfully for some cases, when you need serve specific folder.

### CODE REVIEW

**Comment by greenygh0st** [comment]
The last commit is not required that can be accomplished through the Heroku Procfile. 

Example:

`web: npm install && angular-http-server -p $PORT`

The goal is to keep this as ambiguous as possible.

File: angular-http-server.js [hunk 1]
```diff
@@ -79,3 +78,23 @@ function start() {
         return console.log("Listening on " + getPort());
     });
 }
+
+function returnDistFile() {
+    var path = "index.html";
+    var argvPath = argv.path;
+    if (argvPath && fs.existsSync(argvPath)) {
+        path = argvPath + "/index.html";
```
Comment (dpraul) [inline angular-http-server.js]: Use [`path.join(argvPath, 'index.html')`](https://nodejs.org/api/path.html#path_path_join_paths) so this is OS-agnostic

File: angular-http-server.js [hunk 2]
```diff
@@ -79,3 +78,23 @@ function start() {
...
+        console.log("Can't find %s, using current dir instead", argvPath);
+    }
+
+    return fs.readFileSync(path);
+}
+
+function resolveUrl(filename) {
+    var result = filename;
+    if (filename && argv.path) {
+        result = argv.path + "/" + filename;
```
Comment (dpraul) [inline angular-http-server.js]: Likewise here, use `path.join(argv.path, filename)`

File: angular-http-server.js (hunk 1 above)
Comment (greenygh0st) [reply angular-http-server.js]: 👍

File: angular-http-server.js (hunk 2 above)
Comment (greenygh0st) [reply angular-http-server.js]: 👍

**Comment by simonh1000** [comment]
thanks so much Nick for your efforts. Give me a few days to play with this

**Comment by nick-bogdanov** [comment]
Sure. Thank you too. Give me to know if any problems

**Comment by greenygh0st** [comment]
Any word on when this will be merged @simonh1000? 😄

**Comment by greenygh0st** [comment]
I have a couple of things I am working on that I will need to reconcile with this PR if its merged.

**Comment by simonh1000** [comment]
Thanks everyone - I changed the code style a little, but this was a welcome addition

**Pull request merged**